harness/Makefile

```
@@ -383,18 +385,22 @@ INTEGRATION_ARTIFACTS ?= $(REPO_ROOT)/target/integration
 INTEGRATION_PLAYGROUND_SCENARIO ?= console-streamed-text
 INTEGRATION_PLAYGROUND_ARTIFACTS ?= $(REPO_ROOT)/target/console-e2e
 
-integration-test:
-	@if [ -z "$(III_BIN)" ]; then \
-	  echo "III_BIN is required: path to the pinned iii engine binary."; \
-	  echo "Pinned source: harness/tests/integration/engine.lock"; \
-	  exit 3; \
-	fi
+integration-test: integration-build integration-run
```

checkmake (0.3.2)

[warning] Target "integration-test" should be declared PHONY.

(phonydeclared)